java/java-quiz.md

```diff
@@ -1881,3 +1881,17 @@ numbers                         // Input `List<Integer>` > [1, 2, 3, 4] <br>
  - [ ] out
  - [ ] err
  - [ ] in
+
+#### Q127. The compiler is complaining about this assignment of the variable pickle to the variable jar. How woulld you fix this?
+
```

Comment:
Can you please add the code from the question
![image](https://user-images.githubusercontent.com/13170022/172116666-6a4b6ae2-3e84-489a-a19c-49f549c8636c.png)